### DISCUSSION TITLE
Multiple profiles

### DISCUSSION THREAD

**Discussion by howroyd** [howroyd/simonsays#50] [Ideas]
Requested by [HpXDreamer](https://www.twitch.tv/hpxdreamer):

Multiple profiles so that we can have a normal one with sensible cooldowns then a second, which could be manually switched to with different cooldown settings.

For example, playing normal then someone does a channel point redeem for "chaos mode for 5 mins."  The broadcaster manually switches to the second profile with shorter cooldowns.

**Comment by MorbidMorke** [comment]
This would be fab - it would also be helpful for switching between similar games with slightly different key bindings
